### PR TITLE
Add note about running tests in serial

### PR DESCRIPTION
**Pull request by kentcdodds**
I'm definitely open to changing the wording here.

### CODE REVIEW

**Comment by sindresorhus** [comment]
LGTM

File: readme.md [hunk 1]
```diff
@@ -260,6 +260,7 @@ test.skip('will not be run', t => {
 
 When setup and/or teardown is required, you can use `test.before()` and `test.after()`,
 used in the same manner as `test()`. The test function given to `test.before()` and `test.after()` is called before/after all tests. You can also use `test.beforeEach()` and `test.afterEach()` if you need setup/teardown for each test. Hooks are run serially in the test file. Add as many of these as you want. You can optionally specify a title that is shown on failure.
+If you need to setup some global state between tests using `test.beforeEach()` and `test.afterEach()` (like spying on `console.log` [for example](https://github.com/sindresorhus/ava/issues/560)), you'll need to make sure the tests are run serially (using either [test.serial](#serial-tests) or [`--serial`](#cli)).
```
Comment (vdemedes) [inline readme.md]: Shouldn't it be "set up", not "setup"?

Comment (sindresorhus) [inline readme.md]: Yup

**Comment by sindresorhus** [comment]
Thanks @kentcdodds. Let us know if there's anything else that could be improved with the docs. It's sometimes hard for us to see the obstacles as we're so used to this stuff.

**Pull request closed without merging**